src/NfcIpcSocket.cpp

```diff
@@ -91,37 +90,91 @@ int NfcIpcSocket::GetListenSocket() {
   return nfcdConn;
 }
 
+int NfcIpcSocket::GetConnectedSocket(const char* aSocketName)
+{
+  size_t len = strlen(aSocketName);
+  if (len > (SIZE_MAX - 2)) {
```

Comment:
It represents the leading and trailing '\0' of the socket name. I'll add a constant.